oneview_redfish_toolkit/connection.py

```diff
@@ -138,14 +123,38 @@ def check_oneview_availability(oneview_ip):
 
             if attempt_counter + 1 < attempts:
                 time.sleep(retry_interval_sec)
-        finally:
-            connection.close()
 
     message = "After {} attempts OneView is unreachable at {}".format(
         attempts, oneview_ip)
     raise OneViewRedfishError(message)
 
 
+def request_oneview(oneview_ip, rest_url):
```

Comment:
`test_url` means that is a url to test a connection with oneview? Why not name it just as url?

Reply:
It's the rest url to be requested on OneView, it will be used to test OneView connection and to get appliance node information for service root endpoint as well.